requirements.txt

```diff
@@ -4,3 +4,10 @@ wandb>=0.10.32,<1  # pip will likely update it to 0.12.1, but it is probably ok
 transformers>=4.6.0,<5  # pip will likely update it to 4.10.0, but it is probably ok and good for bugfixes.
 accelerate>=0.4.0,<1  # We may want to use 0.5.0 in the near future
 datasets[streaming]>=1.11.0,<2
+colorama
+konoha
+flair
+segtok
+Pillow>=7.1.0
+nltk
+anyascii
```

Comment:
This seems to me to be similar to REL dependencies. Why do you need to add them here? :blush:

Reply:
Sorry, I fixed it in my latest commit. :) As discussed, I will be merging the PR then.